### PR TITLE
Add locator.Check

### LINKED ISSUE #334: locator.check([options])
The [`locator.check([options])`](https://playwright.dev/docs/api/class-locator#locator-check) method checks the element using the locator's selector with the `strict` option.

### PR DESCRIPTION
**Pull request by inancgumus**
This PR closes #334 .

* Extracts `Frame.check` from `Frame.Check` so that we can use `check` from `Locator.Check`.
* Adds `locator.Check` and a test.
* Renames `strict_link.html` to `locators.html` so that we can use it for all locator tests.

### CODE REVIEW

**Comment by inancgumus** [comment]
GH is experiencing a glitch (although they say all their systems are OK). I hope it's why we get CI failures.

https://www.githubstatus.com/incidents/8xxms3q3qzgp

File: tests/locator_test.go [hunk 1]
```diff
@@ -45,10 +45,36 @@ func TestLocatorDblclick(t *testing.T) {
...
+func TestLocatorCheck(t *testing.T) {
+	tb := newTestBrowser(t, withFileServer())
+	p := tb.NewPage(nil)
+	require.NotNil(t, p.Goto(tb.staticURL("/locators.html"), nil))
+
+	// Selecting a single element and checking it is OK.
+	t.Run("ok", func(t *testing.T) {
+		check := func() bool {
+			cr := p.Evaluate(tb.toGojaValue(`() => window.check`))
+			return cr.(goja.Value).ToBoolean() //nolint:forcetypeassert
```
Comment (imiric) [inline tests/locator_test.go:65]: To get rid of the `nolint` (same for other tests in this file):
```suggestion
			cb, ok := cr.(goja.Value)
			require.True(t, ok)
			return cb.ToBoolean()
```

File: tests/static/locators.html [hunk 2]
```diff
@@ -8,16 +8,22 @@
...
+        window.check = false;
 
         document.querySelector('#link').addEventListener(
             'click', e => { result = true; }, false
         );
         document.querySelector('#link').addEventListener(
             'dblclick', e => { dblclick = true; }, false
         );
+        document.querySelector('#input').addEventListener(
+            'change', e => { window.check = event.currentTarget.checked; }, false
```
Comment (imiric) [inline tests/static/locators.html:25]: How come `event` works here? Shouldn't it be `e`?

**Review by imiric — APPROVED** [review]

File: tests/locator_test.go (hunk 1 above)
Comment (inancgumus) [reply tests/locator_test.go:65]: Alright, I'll address this in one Go for all tests after merging `Check` and `Uncheck`.

File: tests/static/locators.html (hunk 2 above)
Comment (inancgumus) [reply tests/static/locators.html]: Good catch! It works, but to not create confusion in the future, I changed it to `e`: https://github.com/grafana/xk6-browser/pull/362/commits/b1aece296a1db114735651d14dc93b7ccb1122c6

**Pull request merged**